src/App.tsx

```diff
@@ -1,9 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const normalisedQuery = query.toLowerCase().trim();
+  const visibleMovies = moviesFromServer.filter(({ title, description }) => {
+    const normalisedTitle = title.toLowerCase();
+    const normalisedDescription = description.toLowerCase();
+
+    return normalisedTitle.includes(normalisedQuery)
+    || normalisedDescription.includes(normalisedQuery);
```

Comment:
DRY, let us create function for converting to lower case and for `includes` check